src/metabase/driver/generic_sql/query_processor.clj

```diff
@@ -81,7 +81,7 @@
 
   Field
   (formatted [{:keys [schema-name table-name special-type field-name]}]
-    (let [field (keyword (hx/qualify-and-escape-dots schema-name table-name field-name))]
+    (let [field (keyword (hx/qualify-and-escape-dots table-name field-name))]
```

Comment:
same issue. There's a reason for qualifying things with schema names. `schema_1.some_table` is different from `schema_2.some_table` but now we'll just have `some_table` and `some_table`.

If SparkSQL/Drill don't support schemas (`schema-name` is `nil`) the function takes care of that and leaves the schema out.